pallets/committee/src/lib.rs

```diff
@@ -656,6 +657,7 @@ impl<T: Config<I>, I: Instance> ChangeMembers<IdentityId> for Module<T, I> {
         if let Some(curr_rc) = Self::release_coordinator() {
             if outgoing.contains(&curr_rc) {
                 <ReleaseCoordinator<I>>::kill();
+                // TODO: Either remove or use a fixed DID?
                 Self::deposit_event(RawEvent::ReleaseCoordinatorUpdated(
                     Context::current_identity::<Identity<T>>().unwrap_or_default(),
                     None,
```

Comment:
There is no other way to pass in caller details to this code.  So we will need to use a fixed DID here.

Reply:
Lets just drop the DID from the event.